### PR TITLE
random vent smoke explosion thing event is meaner

### PR DESCRIPTION
**Pull request by ghost**
tfw you're walking along a hallway when _YOUR STOMACH CRAMPS HORRIBLY_

##### Changelog

:cl:
rscadd: Some asshole poured a whole beaker of deadly chemicals into the atmospherics scrubbers. Ejection of contents may be deadlier than normal.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
clf3 is like a guaranteed kill if you're unlucky enough to be near a vent, make it like mute toxin or something

**Comment by ghost** [comment]
mute toxin, ok

**Comment by ghost** [comment]
done

**Comment by matskuman5** [comment]
Adam, stop it

**Comment by Slinky5890** [comment]
Not enough title changes, if you make a few more i'll consider agreeing with it

File: code/modules/events/vent_clog.dm [hunk 1]
```diff
@@ -9,8 +9,8 @@
 	endWhen			= 35
 	var/interval 	= 2
 	var/list/vents  = list()
-	var/list/gunk = list("water","carbon","flour","radium","toxin","cleaner","nutriment","condensedcapsaicin","mushroomhallucinogen","lube",
-								 "plantbgone","banana","charcoal","space_drugs","morphine","holywater","ethanol","hot_coco","facid")
+	var/list/gunk = list("mine_salve","unholywater","mindbreaker","uranium","toxin","cleaner","carpotoxin","condensedcapsaicin","mushroomhallucinogen","lube",
```
Comment (Super3222) [inline code/modules/events/vent_clog.dm:12]: im sure glad holywater isn't as OP as it use to be

**Comment by supernoodles13** [comment]
Perhapse laughing gas too? honk!

**Comment by Time-Green** [comment]
"mine_salve","unholywater","mindbreaker","uranium","toxin","cleaner","carpotoxin","condensedcapsaicin","mushroomhallucinogen","lube","plantbgone","mutagen","charcoal","space_drugs","morphine","holywater","ethanol","mutetoxin","facid"
If I had to guess, you got carpotoxin.

**Pull request merged**